data/campaigns/Eastern_Invasion/scenarios/07a_Capturing_the_Ogres.cfg

```diff
@@ -333,55 +355,65 @@
                 side=1
                 race=ogre
             [/filter]
-
             kill=no
             variable=ogres
         [/store_unit]
 
         [switch]
             variable=ogres.length
-
             [case]
                 value=0
-
                 [message]
                     speaker=Gweddry
-                    message= _ "We failed to capture any of them!"
+                    message= _ "Taking slaves doesn’t feel right, even if it is for a good cause. I refuse to capture any of these ogres."
                 [/message]
+                [set_achievement]
+                    content_for=eastern_invasion
+                    id=ei_S07a
+                [/set_achievement]
             [/case]
-
             [case]
                 value=1
-
                 [message]
                     speaker=Gweddry
-                    message= _ "We only captured one ogre."
+                    message= _ "Taking slaves just doesn’t feel right. I only captured one."
+                [/message]
+                [message]
+                    speaker=Dacyn
+                    message= _ "Unfortunate, but at least we have one. Ogres are unskilled fighters but much cheaper to equip than our other soldiers, and thus <i><b>cost much less gold to recall.</b></i>"
```

Comment:
'but are much cheaper'

Reply:
Same for the repeat text below.

Reply:
This phrase is also in S7b.

As an alternative, I think 'Ogres are unskilled fighter but also much cheaper...' works for me.